static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx

```diff
@@ -75,33 +75,33 @@ describe('SpanEvidenceKeyValueList', () => {
 
     const parentSpan = new MockSpan({
       startTimestamp: 0,
-      endTimestamp: 650,
+      endTimestamp: 0.65,
```

Comment:
We should probably change these in the whole file so someone doesn't get bit by copy+paste again, but up to you if you feel like it.

Reply:
done